import.sh

```diff
@@ -30,28 +30,29 @@
 #   # on Linux.
 #
 # Ask @tamird if you run into issues along the way.
-#
-# After committing locally you should run the command below to ensure your repo
-# is in a clean state and then build/test cockroachdb with the new version:
-#   git clean -dxf
 
 set -eu
 
 rm -rf internal/*
-find . -type l -not -path './.git/*' | xargs rm
-curl -sL https://github.com/facebook/rocksdb/archive/v5.0.2.tar.gz | tar zxf - -C internal --strip-components=1
+find . -type l -not -path './.git/*' -exec rm {} \;
+curl -sL https://github.com/facebook/rocksdb/archive/v5.1.2.tar.gz | tar zxf - -C internal --strip-components=1
 make -C internal util/build_version.cc
-# TODO(tamird,petermattis): remove when
-# https://github.com/facebook/rocksdb/commit/4a17b47bb5a2cc09f95acb53ec565bccd171ee4e
-# is released.
-patch -p1 < testharness.patch
+# TODO(tamird): remove when
+# https://github.com/facebook/rocksdb/pull/1910 is merged and release.
+patch -p1 < gettimeofday.patch
 # Downcase some windows-only includes for compatibility with mingw64.
-grep -lRF '<Windows.h>' internal | xargs sed -i~ 's!<Windows.h>!<windows.h>!g'
-grep -lRF '<Rpc.h>' internal | xargs sed -i~ 's!<Rpc.h>!<rpc.h>!g'
+grep -lR '#include <.*[A-Z].*>' internal | while IFS= read -r source_file; do
+	grep -o '#include <.*[A-Z].*>' "$source_file" | while IFS= read -r original; do
+		downcased=$(echo "$original" | tr '[:upper:]' '[:lower:]')
+		sed -i~ "s!$original!$downcased!g" "$source_file"
```

Comment:
This is much easier in `awk`: `awk '/#include/ { print tolower($0); next; } { print; }'`. Unfortunately `awk` doesn't have the handy `-i` option unless we're willing to rely on gnu extensions (and speaking of which, gnu sed has an extension for upper- and lower-casing).